packages/template/src/lib/stack-app/apps/implementations/admin-app-impl.ts

```diff
@@ -150,9 +159,22 @@ export class _StackAdminAppImplIncomplete<HasTokenStore extends boolean, Project
         teamMemberDefaultPermissions: data.config.team_member_default_permissions,
         userDefaultPermissions: data.config.user_default_permissions,
       },
-
+      async getConfig() {
+        return app._adminConfigFromCrud(await app._interface.getConfigOverrides());
+      },
+      // IF_PLATFORM react-like
+      useConfig() {
+        const crud = useAsyncCache(app._configOverridesCache, [], "useConfig()");
+        return useMemo(() => app._adminConfigFromCrud(crud), [crud]);
+      },
+      // END_PLATFORM
       async update(update: AdminProjectUpdateOptions) {
-        await app._interface.updateProject(adminProjectUpdateOptionsToCrud(update));
+        const updateOptions = adminProjectUpdateOptionsToCrud(update);
+        await app._interface.updateProject(filterUndefined({
+          ...updateOptions,
+          config: undefined,
+        }));
+        await app._interface.updateConfigOverrides({ config: updateOptions.config });
         await onRefresh();
       },
```

Comment:
these are calling the wrong endpoints, they should call /config not /config-override